reconcile/rhidp/sso_client/metrics.py

```diff
@@ -0,0 +1,39 @@
+from reconcile.rhidp.metrics import RhIdpBaseMetric
+from reconcile.utils.metrics import (
+    CounterMetric,
+    GaugeMetric,
+)
+
+
+class RhIdpSSOClientReconcileErrorCounter(RhIdpBaseMetric, CounterMetric):
+    """Counter for the failed reconcile runs."""
+
+    @classmethod
+    def name(cls) -> str:
+        return "rhidp_sso_client_reconcile_errors"
+
+
+class RhIdpSSOClientReconcileCounter(RhIdpBaseMetric, CounterMetric):
+    """Counter for successful reconcile runs."""
+
+    @classmethod
+    def name(cls) -> str:
+        return "rhidp_sso_client_reconciled"
+
+
+class RhIdpSSOClientIatExpiration(RhIdpBaseMetric, GaugeMetric):
+    """Gauge for the expiration of the initial access token."""
+
+    path: str
+
+    @classmethod
+    def name(cls) -> str:
+        return "rhidp_sso_client_iat_expiration"
```

Comment:
nit: I find `iat` quite misleading. Especially since jwt appears in this PR context too (iat = issued at). My understanding is `iat` in this context intends to be `initial access token`? Would it make sense to clearly name it or will the metric name be too long then?